site/en/docs/crux/about/index.md

```diff
@@ -0,0 +1,46 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: About CrUX
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Introduction to the CrUX dataset
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  The Chrome User Experience Report (also known as the Chrome UX Report, or CrUX for short) is a dataset that reflects how real-world Chrome users experience popular destinations on the web.
+
+# Required
+date: 2022-06-23
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2020-10-16
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - simonhearne
```

Comment:
Looks like other tooling docs (DevTools, Workbox, Puppeteer) omit the author field. Let's do that here and in the other CrUX pages for consistency. I think having the docs be anonymous is more authoritative.